airflow-code-editor/airflow_code_editor/code_editor_view.py

```diff
@@ -82,7 +82,6 @@ def _save(self, path=None, upload=False):
                         file.write(data.encode('utf-8'))
                         file.write("\n".encode('utf-8'))
                         file.flush()
-                        file.close()
```

Comment:
I was running into an issue with our fork of the `airflow-code-editor` (which still isn't getting pushed to the right subtree! :angry: ), where trying to save a DAG file would result in an error like "Problem loading module: /tmp/tmpfilename.py` (from the next code block). I suspect this was closing the temp file handle before the DAG-validity check could run?

Reply:
Huh.  I suppose it's possible closing may drop the file handle of the temp file, but that surprises me because I did test this before locally and it worked.  I'll test your branch here in a bit to see.  Also I'll check the subtree issue out, I think it's likely just a permission for push issue.

Reply:
Huh!  Your fix is spot on, I don't know how I missed this.  I pushed a hotfix to production, but this should get to master on your branch.